File: 602_1/fourargs.cpp
//Copyright 2018 SichunHao sichunh@bu.edu
#include <iostream>
using namespace std;
int main(int argc,char *argv[])
{

	for(int i=1;i<argc;i=i++)
	{
		if(i>=1 && i<=4)
			{cout<<argv[i]<<endl;
			i=i+1;}
		else if (i>=5)
			{cerr<<argv[i]<<endl;
			i=i+1;}
	}

	return 0;
}

File: 602_1/fourargs.py
#Copyright 2018 SichunHao sichunh@bu.edu
import sys

for i in range(len(sys.argv)):  
	if i<=4 and i>=1:
		sys.stdout.write(sys.argv[i]+'\n')
	elif i>=5:
		sys.stderr.write(sys.argv[i]+'\n')
